src/Documentation/Markdown/utils/remark-linker.js

```diff
@@ -1,23 +1,36 @@
 ;`use strict`
 
 import visit from 'unist-util-visit'
+import { getItemByPath } from '../../SidebarMenu/helper'
 
 function linker() {
   function transformer(tree) {
     visit(tree, 'inlineCode', function(node, index, parent) {
       if (parent.type !== 'link' && /dvc\s+[a-z-.]+/.test(node.value)) {
```

Comment:
do we need to change the regex here to match three-parts nodes?

Reply:
Not necessary. Right now this regexp just check if this is a `dvc` command or not. Breaking it to parts happens at the next line `let parts = node.value.split(/\s+/)`. We can add additional check to `part[2]` to check if it matches command syntax, but we can leave regexp itself as is.